### ISSUE TITLE
'pod install' fails when the 'RCT_NEW_ARCH_ENABLED=1' flag is set

### ISSUE THREAD

**Issue by mfidor** [bamlab/react-native-image-resizer#375]
**Describe the bug**
`pod install` fails when using the `RCT_NEW_ARCH_ENABLED=1` flag.

**To Reproduce**
1. Install `@bam.tech/react-native-image-resizer` npm package in a React Native app.
2. Run the command `RCT_NEW_ARCH_ENABLED=1 npx pod-install`
3. The command fails with the following message:
```
[!] CocoaPods could not find compatible versions for pod "RCT-Folly":
  In Podfile:
    RCT-Folly (from `../node_modules/react-native/third-party-podspecs/RCT-Folly.podspec`)

    React-RCTAppDelegate (from `../node_modules/react-native/Libraries/AppDelegate`) was resolved to 0.72.0, which depends on
      RCT-Folly

    React-RCTVibration (from `../node_modules/react-native/Libraries/Vibration`) was resolved to 0.72.0, which depends on
      RCT-Folly (= 2021.07.22.00)

    react-native-image-resizer (from `../node_modules/@bam.tech/react-native-image-resizer`) was resolved to 3.0.5, which depends on
      RCT-Folly (= 2021.06.28.00-v2)
```

**Expected behavior**
`pod install` should complete successfully when using the `RCT_NEW_ARCH_ENABLED=1` flag with the `@bam.tech/react-native-image-resizer` package.

**Additional context**
Manually changing the `folly_version` to `'2021.07.22.00'` in the `node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec` file allows the `npx pod-install` command to pass. However, this is not a viable solution.

**Environment**
 - `react-native`: `0.72.0`
 - `@bam.tech/react-native-image-resizer`: `3.0.5`

**Comment by ravindraguptacapgemini** [comment]
@mfidor getting the same error, have you found a fix for the issue?

**Comment by ravindraguptacapgemini** [comment]
@chaos2171053 please update if you are releasing a new version this fix?

**Comment by mfidor** [comment]
@ravindraguptacapgemini Sorry, but no. I had issues with other libraries as well, so I'll just stick with the old architecture.

**Comment by taboulot** [comment]
Fix released in [3.0.6](https://github.com/bamlab/react-native-image-resizer/releases/tag/v3.0.6)
